cldfbench_nominalperson_cldf.py

```diff
@@ -20,63 +20,69 @@ def cmd_download(self, args):
 
         >>> self.raw_dir.download(url, fname)
         """
-        pass
+        wals_dir = self.raw_dir / 'wals'
+        if not wals_dir.exists():
+            wals_dir.mkdir()
+        rec = Record.from_doi('10.5281/zenodo.13950591')
+        rec.download_dataset(wals_dir)
 
     def cmd_makecldf(self, args):
         # Extend the schema:
-        args.writer.cldf.add_component('LanguageTable')
-        args.writer.cldf.add_columns('LanguageTable','Walscode')    # add non-standard column
-        
-        args.writer.cldf.add_component('CodeTable')
+        args.writer.cldf.add_component(
+            'LanguageTable',
+            'Walscode',
+            {'name': 'ISOcodes', 'separator': ';', 'datatype': {'base': 'string', 'format': '[a-z]{3}'}})
+
+        #args.writer.cldf.add_component('CodeTable')
         args.writer.cldf.add_component('ParameterTable')
         args.writer.cldf.add_component('ValueTable')
         args.writer.cldf.add_component('ExampleTable')
 
+        glangs = {lg.id: lg for lg in args.glottolog.api.languoids()}
+
+        wals_codes = {r['ID']: r['Name'] for r in self.raw_dir.joinpath('wals').read_csv('codes.csv', dicts=True)}
+        wals_83A = {
+            r['Language_ID']: wals_codes[r['Code_ID']]
+            for r in self.raw_dir.joinpath('wals').read_csv('values.csv', dicts=True)
+            if r['Parameter_ID'] == '83A'}
```

Comment:
We read the WALS values for feature 84A, to compare them to the data in `constitutent_order_wals`.